lighthouse/login.js

```diff
@@ -0,0 +1,108 @@
+const lighthouse = require('lighthouse')
```

Comment:
it looks like all of these files need to be run through prettier.  the current `prettier:fix` package.json script does not include the `lighthouse` dir in the glob pattern.  `{src,cypress,lighthouse}/**/*.{ts,tsx,js}`

Reply:
will add

Reply:
Please let me know if anything else needs to be changed here.